### ISSUE TITLE
Error message can be better when ping bosh director fails

### ISSUE THREAD

**Issue by xiujiao** [genesis-community/genesis#357]
Using genesis v2.6.17, running `genesis deploy env`. failed. The error message is as follows:
```
  # in Genesis::load_yaml_file [u/someone/geese/lib/Genesis.pm:L590]
 TRACE ⮀ From directory: /var/vcap/store/jumpbox/home/someone/concourse-deployments
 TRACE ⮀ Executing: `which "$@" 2>&1`
 TRACE ⮀  - with arguments:
 TRACE ⮀    1: 'ping'
 TRACE ⮀ command exited 0
 VALUE ⮀ run_output = '/bin/ping'
         # in Genesis::tcp_listening [/u/someone/.geese/lib/Genesis.pm:L622]
 TRACE ⮀ From directory: /var/vcap/store/jumpbox/home/someone/concourse-deployments
 TRACE ⮀ Executing: `ping -c "$1" -i "$2" -t "$3" "$4" >/dev/null 2>&1`
 TRACE ⮀  - with arguments:
 TRACE ⮀    1: '2'
 TRACE ⮀    2: '0.2'
 TRACE ⮀    3: '10'
 TRACE ⮀    4: '172.28.158.5'
 TRACE ⮀ command exited with status 1 (rc 0)
 VALUE ⮀ run_output = ''
         # in Genesis::tcp_listening [/u/someone/.geese/lib/Genesis.pm:L623]
unreachable!

[ERROR] Could not connect to BOSH Director 'some-bosh'
  - specified via params.bosh in some-bosh environment file
```
We can run `genesis do env login` for the bosh we are using without problem. but could not run `genesis deploy env`. After debugging, we found out that in `genesis deploy`, it trys to ping the bosh director while in `genesis login`, it does not use ping. While ping is disabled from the vm we run `genesis deploy` to the bosh director which we use to deploy. That is why `genesis deploy` has failed for us while `genesis login` did not.

The error message can be improved to point out the ping failed instead of `Could not connect to BOSH Director.

**Comment by dennisjbell** [comment]
Genesis v2.7.0 no longer uses ping -- it attempts to connect to the correct port over tcp and times out (Defaults to 10s timeout, can use GENESIS_NETWORK_TIMEOUT env var to change)

**Comment by dennisjbell** [comment]
Marking this as resolved because of the changes delivered in 2.7.x branch
